tests/integration/release/test_release.py

```diff
@@ -35,17 +41,117 @@ def test_column_noising(
 
 
 def test_row_noising_omit_row_or_do_not_respond(
-    noised_data: pd.DataFrame,
     dataset_name: str,
     config: dict[str, Any],
 ) -> None:
-    """Tests that omit_row and do_not_respond row noising are being applied"""
-    idx_cols = IDX_COLS.get(dataset_name)
-    original = get_unnoised_data(dataset_name)
-    original_data = original.data.set_index(idx_cols)
-    noised_data = noised_data.set_index(idx_cols)
+    """Tests that the correct noising is applied to each dataset when
```

Comment:
This test is unchanged except removing some tests that will be covered by the split tests.